.circleci/config.yml

```diff
@@ -459,8 +460,12 @@ jobs:
       - run:
           command: |
             cd ${MAGMA_ROOT}/orc8r/cloud/docker
-            python3 build.py --tests --up
+            coverage3 run build.py --tests --up
```

Comment:
Is this just for Python code coverage or for Go as well? Because the `build.py --tests` tests are all Go tests